frontend/src/metabase/css/components/form.css

```diff
@@ -110,14 +110,8 @@
 }
 
 .Form-field:hover .Form-input {
-  color: var(--color-text-light);
-  background: color(var(--color-bg-black) alpha(-98%));
-}
-
-/* ewww */
-.Form-field:hover .Form-input.ng-dirty {
-  color: var(--color-text-dark);
-  background-color: var(--color-bg-white);
+  border-color: var(--color-brand);
+  transition: border 300ms linear;
```

Comment:
Seems like this is only a transition-in without a transition-out. If you put that transition rule on `.Form-field .Form-input` would the border gracefully transition back to gray when you mouse off the input?